src/lib.rs

```diff
@@ -112,6 +116,18 @@ impl Program {
                         return Ok(locals.remove(a as usize));
                     }
                 }
+                Jump(a) => {
+                    iptr += a as isize;
+                    continue;
```

Comment:
Ah good you used the `continue` method, I missed this on my first pass and did it this way in the fixes I just wrote. :)